nginx/config.go

```diff
@@ -128,31 +131,6 @@ http {
 	{{/* This means we force HTTPS if HSTS is enabled. */}}
 	{{ $enforceSecure := or $sslConfig.Enforce $hstsConfig.Enabled }}
 
-	# Default server handles requests for unmapped hostnames, including healthchecks
-	server {
-		listen 8080 default_server reuseport{{ if $routerConfig.UseProxyProtocol }} proxy_protocol{{ end }};
-		listen 6443 default_server ssl {{ if $routerConfig.HTTP2Enabled }}http2{{ end }} {{ if $routerConfig.UseProxyProtocol }}proxy_protocol{{ end }};
-		set $app_name "router-default-vhost";
-		{{ if $routerConfig.PlatformCertificate }}
-		ssl_protocols {{ $sslConfig.Protocols }};
-		ssl_certificate /opt/router/ssl/platform.crt;
-		ssl_certificate_key /opt/router/ssl/platform.key;
-		{{ else }}
-		ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
-		ssl_certificate /opt/router/ssl/default/default.crt;
-		ssl_certificate_key /opt/router/ssl/default/default.key;
-		{{ end }}
-		server_name _;
-		location ~ ^/healthz/?$ {
-			access_log off;
-			default_type 'text/plain';
-			return 200;
-		}
-		location / {
-			return 404;
-		}
-	}
-
```

Comment:
@lachie83 we might want to add this back when DefaultServiceEnabled is false, since removing the default might break healthchecks on existing clusters?